### PR TITLE
almost there!

### PR DESCRIPTION
**Pull request by marcobarilari**
can you try to run one case and take a look at the tsv file to see if we have all the info there?

things to do:

1. target of motion
2. change name of subj.
3. put a delay before the experiment start
4. check what happens when  response is provided
5. other?

### CODE REVIEW

File: getTrialSeq.m [hunk 1]
```diff
@@ -16,56 +16,165 @@
...
-        elseif trial_seq(d) > 3 && trial_seq(d+1) > 3 % avoid 2 consecutive targets
-            d = 1;
-            break
-            % avoid series 3 times the same conditions
-        elseif trial_seq(d) ==  trial_seq(d+1) && trial_seq(d+1) ==  trial_seq(d+2) && trial_seq(d+2) ==  trial_seq(d+3)
-            d = 1;
-            break
-        else
-            d = d+1;
+    case 1
```
Comment (Remi-Gau) [inline getTrialSeq.m:23]: There seems to be a lot of repeated code in there: is it possible to refactor it? Maybe by creating some subfunction within this one.

Aren't case 2 and 3 just the same? Just had a quick look so I might be missing something.

**Review by Remi-Gau — CHANGES_REQUESTED** [review]
the trial generating function needs some refactoring IMHO. ;-)

File: getTrialSeq.m (hunk 1 above)
Comment (marcobarilari) [reply getTrialSeq.m:23]: I refactored a bit and but a subfunction in the function (as we have only one function LMK if we should have a subfun folder)

Comment (Remi-Gau) [reply getTrialSeq.m:23]: much better!

**Review by Remi-Gau — APPROVED** [review]

File: getTrialSeq.m (hunk 1 above)
Comment (Remi-Gau) [reply getTrialSeq.m:23]: Don't think we need a subfolder... Yet

Comment (marcobarilari) [reply getTrialSeq.m:23]: 😂

**Review by CerenB — APPROVED** [review]

**Pull request merged**